vividus/src/main/java/org/vividus/configuration/Vividus.java

```diff
@@ -85,4 +95,26 @@ private static void configureFreemarkerLogger()
         System.setProperty(freemarker.log.Logger.SYSTEM_PROPERTY_NAME_LOGGER_LIBRARY,
                 freemarker.log.Logger.LIBRARY_NAME_SLF4J);
     }
+
+    private static void checkJavaVersion()
+    {
+        if (SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_17))
+        {
+            return;
+        }
+
+        long daysAfterNotification = Duration.between(JAVA_MIGRATION_NOTIFICATION_DATE, LocalDateTime.now()).toDays();
```

Comment:
Shouldn't we run some of the tests on 17? Just to track how it works before the migration date?

Reply:
https://github.com/actions/setup-java#install-multiple-jdks

Reply:
we've been running them for a while: https://github.com/vividus-framework/vividus-sample-tests/tree/main/.github/workflows